packages/stockflux-watchlist/public/preview-chart.html

```diff
@@ -12,6 +12,9 @@
         box-sizing: border-box;
         width: 100vw;
         height: 100vh;
+        background-color:#1B3B56;
+        color:white;
```

Comment:
Nitpick; shouldn't there be a space between the colon and the colour?

Also, can this be a hex value instead of a name to keep things consistent?

Reply:
no longer needed as using the var values from index.css